Pod/Classes/Network/TRYFeedbackUploadManager.m

```diff
@@ -0,0 +1,107 @@
+//
+//  TRYFeedbackUploadManager.m
+//  Pods
+//
+//  Created by Eray on 22/04/16.
+//
+//
+
+#import "TRYFeedbackUploadManager.h"
+#import "TRYFeedbackUploadTask.h"
+
+static NSString * const kStorageDirectoryName        = @"kStorageDirectoryName";
+static NSString * const kTasksKey                    = @"kTasksKey";
+static NSString * const kBackgroundSessionIdentifier = @"kBackgroundSessionIdentifier";
+
```

Comment:
Actual values should be more unique. Let's start them all with `com.hipo.tryouts.`